### ISSUE TITLE
Accessibility issues with the DropDowns module components

### ISSUE THREAD

**Issue by dtopalov** [telerik/kendo-angular#1070]
There are several accessibility issues in some of the components from the DropDowns module (observable when using [ChromeVox extension](https://chrome.google.com/webstore/detail/chromevox/kgejglhpjiefppelpmljglcjbhoiplfn?hl=en)):

- **Autocomplete** - Expanding the list works correctly, but navigating up and down using arrows to go through the items in the list is not registered by the screen reader
- **DropDownList** - Unable to expand the list using the using spacebar key; once expanded, the screen reader correctly reads through the items in the list along with the selected status
 - **MultiSelect** - Expanding the list works correctly, but navigating up and down using the arrows does not read the items in the list (just like in the Autocomplete). Also, the screen reader doesn't read items that are already selected so a non-sighted person wouldn't know which items are already chosen.
- **MultiSelect** - 'Select box' without a description (from WCAG 1.3.1 success criterion). The MultiSelect has both a kendo-search and a kendo-taglist elements inside. The last one contains a list with role "listbox". Because of this, we have to associate a description to the listbox in order to make it accessible. (Reported in Ticket ID: 1154301):

![image](https://user-images.githubusercontent.com/7533599/36479789-a34da600-1712-11e8-90f6-40bcff71b5d5.png)

- **All** - Provide information to the screen reader, when there are no items in the list (through `aria-label`?). Works only when the popup is opened! (Reported in Ticket ID: 1162598)

**Comment by Tesomas** [comment]
We are observing the same issue navigating with the up an down arrows on an expanded combobox.

**Comment by RobertBruening** [comment]
We are experiencing the same issue with not being able to navigate an expanded comboBox with up/down arrows

**Comment by gchurchley** [comment]
We are using the multiselect component in production and our clients are complaining about the accessibility issues (most of which have been identified in this issue).
Non-sighted users are unable to use the MultiSelect component.  This is a very important issue for us. We use ChromeVox and NVDA on all modern browsers.  When can we expect a fix for these accessibility issues?  Thank you.

**Comment by tsvetomir** [comment]
Apologies for the delay, everyone. We'll review the accessibility of the drop-down components and will get back to you.

**Comment by valchev** [comment]
Hello guys, the latest dev version of the package resolves the following issues:

- AutoComplete, ComboBox & MultiSelect: Expanding the list works correctly, but navigating up and down using arrows to go through the items in the list is not registered by the screen reader.

- MultiSelect: the screen reader doesn't read items that are already selected so a non-sighted person wouldn't know which items are already chosen.

- All: Provide information to the screen reader, when there are no items in the list (through aria-label). Works only when the popup is opened!

- MultiSelect: The search-bar subcomponent is described by tag-list subcomponent via `aria-describedby` attribute

**Comment by ipeshev** [comment]
Regading expanding dropdownlist using space bar , by specification it should be done with ALT+ARROW_DOWN.

**Comment by ssmartin** [comment]
Hi, 

Is the accessibility problem with the tag-list solved? 

> MultiSelect - 'Select box' without a description (from WCAG 1.3.1 success criterion). The MultiSelect has both a kendo-search and a kendo-taglist elements inside. The last one contains a list with role "listbox". Because of this, we have to associate a description to the listbox in order to make it accessible. (Reported in Ticket ID: 1154301):


I just took the latest dev dropdown version ("@progress/kendo-angular-dropdowns": "^3.4.0-dev.201811160917",) and I still get the warning using siteimprove. 

![image](https://user-images.githubusercontent.com/17272626/48696714-60486800-ebeb-11e8-908f-4d55ef1ebee3.png)

Should I create a new support ticket for this specific issue? 


Thank you.

**Comment by ssmartin** [comment]
HI @valchev ,

Checked the github repository and indeed the fix was made only for searchbar component, not for the tag-list. is it planned to fix also the tag-list or I should create a support ticket from our telerik account? 

Please advise,
Thank you.

**Comment by danielkaradachki** [comment]
Sorry for the delayed reply. We have opened a separate issue for the taglist:

https://github.com/telerik/kendo-angular/issues/1939
